src/js/components/DateInput/utils.js

```diff
@@ -125,6 +125,30 @@ const pullDigits = (text, index) => {
   return text.slice(index, end);
 };
 
+export const validateDateBounds = ({ dateBounds, selectedDate }) => {
+  if (!dateBounds || !selectedDate) return selectedDate;
+
+  const [startDate, endDate] = dateBounds.map((date) =>
+    new Date(date).toISOString(),
+  );
+
+  const selectedDateArray = Array.isArray(selectedDate)
```

Comment:
This is a small recommendation to simplify having to declare so many variables as well as small naming suggestions:

```
  const isoSelectedDate = (
    Array.isArray(selectedDate) ? selectedDate : [selectedDate]
  ).map((date) => new Date(date).toISOString());
```

Reply:
This would all you to remove isoSelectedDateArray and just consolidate here.

Reply:
changed